omnigibson/object_states/saturated.py

```diff
@@ -2,15 +2,17 @@
 
 from omnigibson.macros import create_module_macros
 from omnigibson.object_states.object_state_base import BooleanStateMixin, RelativeObjectState
-from omnigibson.systems.system_base import REGISTERED_SYSTEMS, UUID_TO_SYSTEMS
-from omnigibson.utils.python_utils import get_uuid
+from omnigibson.utils.python_utils import StringIntegerMapper
 
 # Create settings for this module
 m = create_module_macros(module_path=__file__)
 
 # Default saturation limit
 m.DEFAULT_SATURATION_LIMIT = 1e6
 
+# Create string to integer mapping
+m.NAME_MAPPER = StringIntegerMapper()
```

Comment:
Should this be a macro? I don't think it's a hardcoded value

Reply:
This definitely shouldn't be a macro, but it probably shouldn't even be a global variable. Please discuss this tomorrow - I think this could cause plenty of issues with respect to deterministic save/loads between different versions of OG etc. and it would not respect clear calls etc